### ISSUE TITLE
.resume() is not a function error when calling resume on a Sound object.

### ISSUE THREAD

**Issue by nadav2051** [zmxv/react-native-sound#610]
:beetle: **Description**

calling .resume() produces a 'resume is not a function' error.

:beetle: **What is the observed behavior?**

A Sound reference has been initialized, a play has been called on that reference, the music has started playing.
Once a .resume() has been called on the same reference, RN threw an error of 'resume is not a function'.
When switching .resume() to .play() at the exact same location, it's working properly, the only problem is it's playing the song from the beginning. however that negates the possibility I'm working on a null reference in that state.

:beetle: **What is the expected behavior?**

I expect the sound to resume at the same time I've called the .pause() function.

:beetle: **Please post your code:**

```js
    onPressReadyToPlayState() {
        this.state.soundReference.play();
        this.setState({
            playStatus: SoundStatusEnum.PLAYING
        });
    }

    onPressPlayingState() {
        this.state.soundReference.stop();
        this.setState({
            playStatus: SoundStatusEnum.PAUSED
        });
        return;
    }

    onPressPausedState() {
        this.state.soundReference.resume();
        this.setState({
            playStatus: SoundStatusEnum.PLAYING
        });
        return;
    }```

:bulb: **Does the problem have a test case?**

<!--- Can you reproduce the problem in a new project? (`react-native init`)?
If so, please post a link to the project. --> 

:bulb: **Possible solution**

<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

:bulb: **Is there a workaround?**

<!--- Until the problem is fixed, is there a way to work around it? -->

:bulb: **If the bug is confirmed, would you be willing to create a pull request?**

<!--- Help can be provided if you need assistance creating a PR. -->

**Is your issue with...**

- [ X ] iOS
- [ Haven't tried yet ] Android 
- [ Will not try ] Windows

**Are you using...**

- [ ] React Native CLI (e.g. `react-native run-android`)
- [ ] Expo
- [ X ] Other: Our own flavour of the native app that mounts RN and it's components

**Which versions are you using?**

- React Native Sound: 0.11.0
- React Native: 0.60.4
- iOS: 12.3
- Android: Not relevant
- Windows: Not relevant

**Does the problem occur on...**

- [ X ] Simulator
- [ ] Device

**Comment by paulmelnikow** [comment]
Could you post a stack trace?

**Comment by nadav2051** [comment]
Sure, I'll reproduce and post it, but I've also noticed that I can't find .resume() in the API representation at the index.d.ts, is it possible it has been deprecated?

**Comment by paulmelnikow** [comment]
I don't see a `.resume()` function. I think you just use `.play()`.

**Comment by nadav2051** [comment]
Yep, I've worked around by tracking current second while pausing, and seek to position before playing again.

just .play()ing will restart the audio.

Anyway, I'd say the API on the front page should be updated to remove the Resume functionality unless we've both missed something.

**Comment by meghaptg** [comment]
> Yep, I've worked around by tracking current second while pausing, and seek to position before playing again.
> 
> just .play()ing will restart the audio.
> 
> Anyway, I'd say the API on the front page should be updated to remove the Resume functionality unless we've both missed something.

Hey @nadav2051 may I know what the workaround was?

**Comment by nadav2051** [comment]
@meghaptg 
Sure!

One you start playing, you set an interval that tracks the seconds played (**soundObject.getCurrentTime**), keep that data in the object level, and when a time when you want to resume comes, use **soundObject.setCurrentTime** to the time you have saved from the interval tracking the getCurrentTime, followed by **soundObject.play()**.

Hope you found this helpful!
